tasmota/tasmota.ino

```diff
@@ -23,8 +23,8 @@
 #endif  // ESP32_STAGE
 #include "include/tasmota_compat.h"
 #include "include/tasmota_version.h"        // Tasmota version information
-#include "include/tasmota.h"                // Enumeration used in my_user_config.h
 #include "my_user_config.h"                 // Fixed user configurable options
+#include "include/tasmota.h"                // Enumeration used in my_user_config.h
```

Comment:
I'm concerned that changing the include order has much more impact and may break many stuff. Can you please preserve the include order?

Reply:
Hi, the new order of the includes is needed for this change to work. I already tested and there is no issue with that since **my_user_config.h** is just a lot of macro definitions which don't change anything on **tasmota.h** (except for the default value of WIFI HOSTNAME that this PR changes.